ArmTemplates/apim-api-resources.json

```diff
@@ -20,6 +20,13 @@
         "description": "The hostname used by the API (Proxy) and Developer Portal (Portal)"
       }
     },
+    "LoggerSamplingPercentage": {
+      "type": "int",
+      "defaultValue": "100",
```

Comment:
Suggest adding min and max values to catch any daft value being passed in
```"minValue": 0,
"maxValue": 100,```

The documentation doesnt mention what the allowed values are so I've gone with the obvious min and max